src/components/Wiki/Entry.tsx

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
+import {Tab, Tabs, Card, Form, Button, Image, Toast, Modal} from "react-bootstrap";
+
+import './Entry.css';
+
+const logo = require('../../images/swarmLogoIcon.png');
+
+/*
+* There's a warning that gets produced in the console when interacting with the Tab component below (switching b/t details/comment/details tabs)
+* react-bootstrap is aware of this warning but they do not have a fix for it yet it seems.
+*
+* warning:
+* "findDOMNode is deprecated in StrictMode..."
+*
+* This can be suppressed by passing {false} to the 'transition' prop in the Tabs/Tab components.
+* So no fancy fade in/out transitions between tabs :(
+* */
+
+interface entryState{
+    replyModalShow: boolean
+    replyModalQuote: string
+}
+
+interface entryProps{
+    id: string
+}
+
+//component has no props, hence {}
+class Entry extends React.Component<entryProps, entryState>{
+    constructor(props: entryProps) {
+        super(props);
+        this.state = {
+            replyModalShow : false,
+            replyModalQuote: "test"
+        };
+        this.handleHide = this.handleHide.bind(this);
+        this.handleShow = this.handleShow.bind(this);
```

Comment:
This isn't for a fix but you can actually bypass having to use these declaration if you use arrow functions instead in the given methods. You won't need to do binds here if you do.